packages/components/src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, PropsWithChildren } from "react";
+import * as Aria from "react-aria-components";
+import styles from "./Switch.module.css";
+import clsx from "clsx";
+
+export interface SwitchProps
+  extends PropsWithChildren<Omit<Aria.SwitchProps, "children">> {
+  /** @default "indicatorFirst" */
+  order: "indicatorFirst" | "labelFirst";
```

Comment:
Klingt für imo iwie "holprig". Wie wäre es mit

```suggestion
  labelPosition: "leading" | "trailing";
```